index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Kelvin &mdash;portfolio</title>
+</head>
+<body>
+
+<nav class="nav_container">
+    <h3>Kely Bush</h3>
+    <img src="assets/nav-icon.svg" alt="Toggle Menu">
+</nav>
```

Comment:
- [ ] Good job working on the navigation bar. You are required to place your navbar in a `<header>` tag and the logo should be in an `<a>` tag. Please fix this issue.
- [ ] Please place your logo text outside of `<nav>` tag.